### PR TITLE
setting-up-coin-card

### PR DESCRIPTION
**Pull request by jvin08**

### CODE REVIEW

File: app/components/coinPortfolioItem/CoinCard.tsx [hunk 1]
```diff
@@ -49,23 +52,23 @@ const CoinCard = ({ storageData, toggleDeleteModal, toggleEditModal }:{ storageD
             </div>
             <div className="flex justify-between pb-4 mt-5">
               <DataElement name="Current price:" value={coin.currentPrice} width="w-1/12" />
-              <DataElement name="Price cgange 24h:" value={coin.priceChange} width="w-1/3" />
+              <DataElement name="Price cgange 24h:" value={coin.priceChange} width="w-1/3 ml-auto" />
```
Comment (yoloOnTheBattlefield) [inline app/components/coinPortfolioItem/CoinCard.tsx]: change

File: app/components/coinPortfolioItem/DataElement.tsx [hunk 2]
```diff
@@ -3,10 +3,11 @@ import React from "react";
 import clsx from "clsx";
 import { useSelector } from "react-redux";
 import { selectDarkmode } from "@/app/lib/dynamicValuesSlice";
+import { formatTime } from "./utils";
 
-const DataElement = ({name, value, width}:{name: string, value: string, width: string}) => {
+export const DataElement = ({name, value, width}:{name: string, value: string, width: string}) => {
   const darkmode = useSelector(selectDarkmode);
-  const formattedValue = Math.abs(Number(value));
+  const colorRed = Number(value) < 0 && true;
```
Comment (yoloOnTheBattlefield) [inline app/components/coinPortfolioItem/DataElement.tsx]: const isNegative = Number(value) < 0;

File: app/components/coinPortfolioItem/DataElement.tsx [hunk 3]
```diff
@@ -16,10 +17,22 @@ const DataElement = ({name, value, width}:{name: string, value: string, width: s
         <p className="text-[11px] mb-1">{name}</p>}
       <p className={clsx("text-xs",{
         "text-cryptoblue-650": darkmode,
-        "text-cryptoblue-660": Number(value) > 0 && !darkmode,
-        "text-cryptoblue-750": Number(value) < 0,
-      })}>{formattedValue}</p>
+        "text-cryptoblue-660": !darkmode,
+        "text-cryptoblue-750": colorRed,
```
Comment (yoloOnTheBattlefield) [inline app/components/coinPortfolioItem/DataElement.tsx]: isNegative

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: app/components/coinPortfolioItem/DataElement.tsx (hunk 3 above)
Comment (jvin08) [reply app/components/coinPortfolioItem/DataElement.tsx]: fixed, in next commit

File: app/components/coinPortfolioItem/DataElement.tsx (hunk 2 above)
Comment (jvin08) [reply app/components/coinPortfolioItem/DataElement.tsx]: fixed, in next commit

File: app/components/coinPortfolioItem/CoinCard.tsx (hunk 1 above)
Comment (jvin08) [reply app/components/coinPortfolioItem/CoinCard.tsx]: changed in following commit

**Pull request merged**